app/src/main/AndroidManifest.xml

```diff
@@ -62,7 +65,21 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-    </application>
 
```

Comment:
Why do you need to make so many fake activities?

Reply:
Since we can't inject dependencies on the regular activities, I made mock versions of it, using other local databases, to test it.